cli-output/src/cli_output.rs

```diff
@@ -686,7 +701,15 @@ pub struct CliStakeState {
 }
 
 impl QuietDisplay for CliStakeState {}
-impl VerboseDisplay for CliStakeState {}
+impl VerboseDisplay for CliStakeState {
+    fn write_str(&self, w: &mut dyn std::fmt::Write) -> std::fmt::Result {
+        write!(w, "{}", self)?;
+        if let Some(credits) = self.credits_observed {
+            writeln!(w, "Credits Observed: {}", credits)?;
+        }
```

Comment:
Since this is a verbose field, would it make sense to display it unconditionally? `.unwrap_or(0)`

Reply:
I went back and forth on that, and I think I'm perfectly ambivalent, so I'll let you nudge me over the edge :)

Reply:
Actually, that's not quite right because of the Uninitialized and RewardsPool cases. But I will include on Initialized: https://github.com/solana-labs/solana/pull/13923/commits/e0b37be17517c05591f71405b3543f1c3c2e18f3